pennylane/transforms/core/transform_dispatcher.py

```diff
@@ -521,17 +568,25 @@ def classical_cotransform(self) -> None | Callable:
 
     @property
     def plxpr_transform(self) -> None | Callable:
-        """The stored quantum transform's PLxPR transform."""
+        """**UNMAINTAINED EXPERIMENTAL:** The stored quantum transform's PLxPR transform."""
```

Comment:
Is this the cause of Sphinx's `Warning: Inline strong start-string without end-string. [docutils]`?